packages/atlas/src/api/client/cache.ts

```diff
@@ -124,6 +125,15 @@ const queryCacheFields: CachePolicyFields<keyof Query> = {
       })
     )
   },
+  ownedNftByUniqueInput: (existing, { toReference, args }) => {
+    return (
+      existing ||
+      toReference({
+        __typename: 'OwnedNft',
+        id: args?.where.id,
+      })
+    )
+  },
```

Comment:
Why do we need that?

Reply:
To reduce the number of requests to Orion. Without this chunk of code, I was fetching all the NFTs using `useChannelNfts`, and then fetching them again for every tile using `useNft`(which is called inside NftTileViewer).

Reply:
Okay, makes sense!